src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -1438,10 +1461,11 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
                 y: boundingBox[i + 1],
             });
         }
-
+        const regionTag = this.props.project.tags.filter((tag) => tag.name === tagName);
```

Comment:
use find, we only have one tag with the given name

Reply:
Is there any case tagName can be empty or cannot find a tag?

Reply:
this function would be called in convertLabeldataToRegion, therefore, I don't think there will be any cases, we have a ghost tag in labeldata